diesel/src/query_source/aliasing/macros.rs

```diff
@@ -79,23 +79,23 @@ macro_rules! alias {
     ($($($table: ident)::+ as $alias_name: ident: $alias_ty: ident),* $(,)?) => {
         $crate::alias! {
             $(
-                const $alias_name: Alias<$alias_ty> = $($table)::+ as $alias_name;
+                pub const $alias_name: Alias<$alias_ty> = $($table)::+ as $alias_name;
             )*
         }
     };
-    ($(const $const_name: ident: Alias<$alias_ty: ident> = $($table: ident)::+ as $alias_sql_name: ident);* $(;)?) => {
-        $crate::alias!(NoConst $($($table)::+ as $alias_sql_name: $alias_ty,)*);
+    ($($vis: vis const $const_name: ident: Alias<$alias_ty: ident> = $($table: ident)::+ as $alias_sql_name: ident);* $(;)?) => {
```

Comment:
Just to make that sure: We have a 
```
diesel::alias!{
     const USERS_ALIAS_2: Alias<UsersAlias2> = schema::users as users_alias_2;
}
```
like case somewhere to verify that we could create private variants too?

Reply:
almost: we've got the in-function version of the macro:
```rust
($($($table: ident)::+ as $alias: ident),* $(,)?) => {{
        $crate::alias!(NoConst $($($table)::+ as $alias: $alias,)*);
        ($($crate::query_source::Alias::<$alias>::default()),*)
}};
```
that does not pass the visibility modifier arg ([which is always optional](https://doc.rust-lang.org/reference/macros-by-example.html#metavariables)), so we know the macro is able to parse that properly.

Reply:
I'm going to add that to the visibility test anyway, that's quickly done